client/webserver/site/src/js/charts.js

```diff
@@ -818,6 +833,9 @@ export class CandleChart extends Chart {
 
     // Draw the volume bars.
     const volDataExtents = new Extents(start(first), end(last), 0, highVol)
+    if (!this.volumeRegion) {
+      this.setRegions()
```

Comment:
As you suggested in the PR description, this probably isn't needed for the candle chart.  I bet this will be impossible to hit because `CandleChart`'s constructor calls `resize` (and then `resized` where `volumeRegion` is set), but this seems fine.

Reply:
Just removed since probably not needed.